pilot/pkg/proxy/envoy/testdata/destination-google-timeout-v1alpha2.yaml.golden

```diff
@@ -0,0 +1,19 @@
+name: "*.google.com"
+trafficPolicy:
+  loadBalancer:
+    simple: RANDOM
+  connectionPool:
+    tcp:
+      # TODO: connectTimeout: 3s
+      maxConnections: 100
+    http:
+      http1MaxPendingRequests: 50
+      http2MaxRequests: 200
+      maxRequestsPerConnection: 500
+      maxRetries: 5
+  outlierDetection:
+    http:
+      consecutiveErrors: 10
+      interval: 30s
+      baseEjectionTime: 15.5s
+      maxEjectionPercent: 100
```

Comment:
where is timeout here? This seems to be CB as the earlier one

Reply:
ah, nice catch. timeouts were moved to routing rules, this file should be removed.